docs/app/pages/components/components-sections/select/select/snippets/app.html

```diff
@@ -75,6 +76,9 @@
                         <ux-checkbox [value]="pagingEnabled | async"
                             (valueChange)="pagingEnabled.next($event)">Enable Option Paging</ux-checkbox>
                     </div>
+                    <div class="col-md-4 col-sm-12">
+                        <ux-checkbox [(value)]="clearButton" [disabled]="!(multiple | async) && !allowNull">clearButton</ux-checkbox>
+                    </div>
```

Comment:
Can we shift this to be adjacent to the multiple/disabled/allowNull/readonlyInput checkboxes? Add a new row as needed. This section is intended to have some vaguely logical grouping, with the boolean inputs appearing together.